pkg/bpftrace/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+# TODO: chance once PR #1 has been merged
+FROM lfedge/eve-alpine:de974c9d8b7bc1f30e7a6a062e91271b1e9e7236 AS build
+#FROM eve-alpine AS build
+
+ENV BUILD_PKGS make gcc g++ perl linux-headers musl-dev cmake zlib-dev bcc-dev libbpf-dev cereal flex bison llvm13-libs llvm13-dev llvm13-static clang-dev clang-static pahole gtest-dev bash
+
+RUN eve-alpine-deploy.sh
+
+RUN mkdir -p /usr/src
+ADD https://github.com/christoph-zededa/bpftrace/archive/refs/heads/eve.zip /usr/src
```

Comment:
Is there a standard place to pull this from an repo upstream so that we can get a version string etc?

Reply:
I switched to the official repository and putting my code as patches on top.